README.md

```diff
@@ -1,6 +1,16 @@
 ![MobiMart](https://socialify.git.ci/kavania2002/MobiMart/image?description=1&descriptionEditable=MobiMart%20is%20a%20mobile%20shopping%20website.%20Very%20useful%20for%20Django%20beginner.&font=Bitter&forks=1&issues=1&language=1&owner=1&pattern=Plus&pulls=1&stargazers=1&theme=Light)
 <br>
 
+# MobiMart
+<br>
+
+![Main Page ss1](https://user-images.githubusercontent.com/78464111/107855526-d4186000-6e48-11eb-9364-35e753427f8e.png)
+<br>
+![Shopping Cart ss](https://user-images.githubusercontent.com/78464111/107855564-32454300-6e49-11eb-838a-0bbf5c13146d.png)
```

Comment:
Why the tab of logout and login showing simultaneously. See that it is running properly. You can add screenshot from the website given.

Reply:
Ok i will check that